BaseClasses.py

```diff
@@ -1117,6 +1032,8 @@ def __init__(self, name: str, advancement: bool, code: Optional[int], player: in
         self.player = player
         self.code = code
 
+        self.can_be_excluded = not advancement
```

Comment:
exlusion update and minecraft update are two distinct topics and would be preferred as two pull requests.